verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -179,6 +205,8 @@ bool aws_cryptosdk_hdr_members_are_bounded(
 
 enum aws_cryptosdk_sha_version aws_cryptosdk_which_sha(enum aws_cryptosdk_alg_id alg_id) {
     switch (alg_id) {
+        case ALG_AES256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384:
+        case ALG_AES256_GCM_HKDF_SHA512_COMMIT_KEY: return AWS_CRYPTOSDK_SHA512;
```

Comment:
Instead of redefining this here, can we expose the implementation in cipher.c in the private/cipher.h header, and use that in the proofs?

Reply:
That would be great if it could be done. I had a problem with `which_sha` and then I realized our version has not been updated, but we should avoid this kind of tricks whenever possible.

Reply:
And yes, we can also verify the other case. I will update tomorrow!